app/src/components/annotation/AnnotationInputExplanation.tsx

```diff
@@ -1,65 +1,99 @@
 import React from "react";
 import { FocusScope } from "react-aria";
-import { Label, TextArea } from "react-aria-components";
-
-import { Tooltip, TooltipTrigger, TriggerWrap } from "@arizeai/components";
+import { css } from "@emotion/react";
 
 import {
   Button,
   Dialog,
   DialogTrigger,
   Flex,
+  Input,
+  Label,
   Popover,
   PopoverArrow,
   Text,
   TextField,
   View,
 } from "@phoenix/components";
-import { Icon, Icons } from "@phoenix/components/icon";
+import { Annotation } from "@phoenix/components/annotation";
+
+export const EXPLANATION_LABEL_WIDTH = "44px";
 
 export const AnnotationInputExplanation = ({
-  explanation,
+  annotation,
   onSubmit,
+  containerRef,
 }: {
-  explanation?: string;
+  annotation?: Annotation;
   onSubmit?: (explanation: string) => void;
+  containerRef?: HTMLDivElement;
 }) => {
+  const fieldName = annotation?.name
+    ? `${annotation.name}.explanation`
+    : "explanation";
   return (
     <DialogTrigger>
-      <TooltipTrigger placement="top">
-        <TriggerWrap>
-          <Button variant="quiet" type="button">
-            <Icon svg={<Icons.Edit2Outline />} />
-          </Button>
-        </TriggerWrap>
-        <Tooltip>Explain this score</Tooltip>
-      </TooltipTrigger>
-      <Popover placement="bottom end">
+      <Button
+        excludeFromTabOrder
+        type="button"
+        isDisabled={!annotation?.id}
+        className="annotation-input-explanation"
+        css={css`
```

Comment:
might be just worth using the aria-button rather than fight the css?

Reply:
Going to punt on this for now, was overriding enough stuff that it felt worth it